src/components/v2v3/V2V3Project/V2V3FundingCycleSection/ReservedTokensSplitsCard.tsx

```diff
@@ -82,7 +82,7 @@ export default function ReservedTokensSplitsCard({
                 paragraph={{ rows: 1, width: 20 }}
                 title={false}
               >
-                <span className="text-base font-medium">
+                <span className="text-primary text-base font-medium">
```

Comment:
**note (nonblocking)** not your fault, but `text-primary` should be either deprecated, or updated to use tailwind somehow. Noting so we can follow up later.